specs/PointOfInterest/WeatherStation/generate.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/python3
```

Comment:
please document here what this program is doing

Reply:
Fixed in https://github.com/FIWARE/dataModels/pull/386/commits/939738767c0ec7f2e36bf9376fc3014e0fffc0e7